src/Workspaces/Core/Portable/SymbolKey/SymbolKey.ErrorTypeSymbolKey.cs

```diff
@@ -13,9 +15,23 @@ private static class ErrorTypeSymbolKey
             public static void Create(INamedTypeSymbol symbol, SymbolKeyWriter visitor)
             {
                 visitor.WriteString(symbol.Name);
-                visitor.WriteSymbolKey(symbol.ContainingSymbol as INamespaceOrTypeSymbol);
-                visitor.WriteInteger(symbol.Arity);
+                switch (symbol.ContainingSymbol)
+                {
+                    case INamedTypeSymbol parentType:
+                        visitor.WriteInteger(0);
+                        visitor.WriteSymbolKey(parentType);
```

Comment:
We can't get cases where the parent is a type that's _also_ in a containing namespace?

Reply:
not sure what you mean. will ping you

Reply:
Ah I was worried about the case where an error type symbol is parented in another error type symbol that itself is parented with error namespaces. I guess this is fine since the WriteSymbolKey isn't assuming it's not an error type symbol.

Reply:
got it :)